### DISCUSSION TITLE
Thank you and Happy new year!

### DISCUSSION THREAD

**Discussion by mariusa** [fastify/fastify#3582] [General] [answered]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the issue has not already been raised

### Issue

Dear Matteo & team, 

Just wanted to say Thank you for a professional set of packages, critical for node web app development! And thanks for the awesome docs and support provided every time :)

Happy new year!

**Comment by mcollina** [comment, marked as answer]
Thank you @mariusa!
